### DISCUSSION TITLE
Assessment of the difficulty in porting CPU architecture for magma

### DISCUSSION THREAD

**Discussion by wangyuliu** [magma/magma#15342] [Communications]
Hello everyone! I am working on implementing a tool to assess the complexity of CPU architecture porting. It primarily focuses on RISC-V architecture porting. In fact, the tool may have an average estimate of various architecture porting efforts.My focus is on the overall workload and difficulty of transplantation in the past and future,even if a project has already been ported.As part of my dataset, I have collected the **magma** project. **I would like to gather community opinions to support my assessment. I appreciate your help and response!** Based on scanning tools, the porting complexity is determined to be simple, with a small amount of code related to the CPU architecture in the project.  Is this assessment accurate?Do you have any opinions on personnel allocation and consumption time？ I look forward to your help and response.

**Comment by panyogesh** [comment]
This will be very specific to the CPU architecture. Majority of the testing is being done on Intel and some in Arm.  I would say this would become little off-topic for discussion but please join the town-hall meetings and share the feedback. Collectively we can decide on the way forward,
